spec/services/disclosure_log_spec.rb

```diff
@@ -40,7 +40,10 @@
                       publishable: true }),
        double(reference: 'FOI-4',
               to_h: { reference: 'FOI-4',
-                      publishable: false })]
+                      publishable: false }),
+       double(reference: 'FOI-999',
+              to_h: { reference: 'FOI-999',
```

Comment:
Need to update with a source after https://github.com/mysociety/foi-for-councils/pull/406 is merged.